synapse/lib/httpapi.py

```diff
@@ -1344,6 +1344,7 @@ async def _runHttpExt(self, meth, path):
         varz['_http_request_info'] = info
 
         opts = {
+            'mirror': adef.get('pool', False),
```

Comment:
Do we want to do a migration to set this value?

Reply:
If there was more than one place where this value was used, I'd say yes but since it's easily handled with the default value parameter, a migration probably isn't needed.

Reply:
We'll address it in a future release. The schema validation will automatically set it if it is not present upon modificaitons, so it may be moot until we have to consider a 3.0.0 migration.